lib/node_modules/@stdlib/blas/ext/base/dapxsumpw/examples/c/example.c

```diff
@@ -25,10 +25,10 @@ int main( void ) {
 	const double x[] = { 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 };
 
 	// Specify the number of elements:
-	const int64_t N = 4;
+	const int N = 4;
```

Comment:
@headlessNode For future reference, this change is okay, but also unnecessary.

Reply:
...but upon thinking about it, I see your rationale, so if you want to keep making this change, that is okay, as well.